scratch/youLike.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python
```

Comment:
This script looks good :+1: 

If I HAD to point out something, I'd say use something like [black](https://github.com/psf/black) to format the code.